.gitignore

```diff
@@ -3,6 +3,8 @@ test.ini
 what.ini
 cookies.jar
 *.json
+!user_*.json
+!track*.json
```

Comment:
I'd prefer `!test/fixtures/*.json` as that makes it more explicit what's going on.